### PR TITLE
refactor: further clean up python data model & introduce function definition

### PR DESCRIPTION
**Pull request by bossenti**
### Purpose

This PR contains some minor refactorings:
* improve import paths of `broker`-classes
* restructure python model and reduce usage of `BaseElement` class for a more concise class hirarchy

Furthermore, it introduces the data model of a `FunctionDefinition` which is required for the further extension of the function handler.

### Remarks

PR introduces (a) breaking change(s): no
PR introduces (a) deprecation(s): no

### CODE REVIEW

**Comment by SvenO3** [comment]
I also work on the python functions at the moment and try to support output data streams like in Java (see #1135). So I am changing the data model and definition of a function too, but I try to fit my changes with yours.

File: streampipes-client-python/streampipes_client/model/common.py [hunk 1]
```diff
@@ -55,30 +55,33 @@ class BaseElement(BasicModel):
...
 
-class EventPropertyQualityRequirement(BaseElement):
+class EventPropertyQualityRequirement(BasicModel):
     """
     Data model of an `EventPropertyQualityRequirement` in compliance to the StreamPipes Backend.
     """
 
+    element_id: Optional[StrictStr]
     minimum_property_quality: Optional[BaseElement] = Field(alias="eventPropertyQualityDefinition")
     maximum_property_quality: Optional[BaseElement] = Field(alias="eventPropertyQualityDefinition")
```
Comment (SvenO3) [inline streampipes-client-python/streampipes_client/model/common.py:65]: The EventPropertyQualities and EventPropertyQualityRequirement aren't used by the API anymore and I think we could remove them. The elementId is also not used for every class anymore and we probably should change this. But I can do this in another PR too.

**Review by SvenO3 — APPROVED** [review]

File: streampipes-client-python/streampipes_client/model/common.py (hunk 1 above)
Comment (bossenti) [reply streampipes-client-python/streampipes_client/model/common.py:65]: That would be great 👍

**Comment by bossenti** [comment]
@SvenO3 cool 🤩 
Then let's align on this somewhen in the near future

**Pull request merged**